styles.css

```diff
@@ -538,31 +531,207 @@ h1.heading {
   margin-bottom: 8px;
 }
 
-@media screen and (max-width: 768px) {
-  .container_small {
-    padding-left: 8px;
-    margin-top: -60px;
+.hand {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  body {
+    background-color: rgb(240, 240, 240);
+  }
+
+  .material {
+    background-color: #fff;
   }
 
   .section {
+    padding: 0 3%;
     background-image: url(asset/header-shapes-mobile.svg);
     background-repeat: no-repeat;
     background-size: cover;
+    border-bottom-left-radius: 100px;
+    background-color: #fff;
   }
 
-  .toolbar {
-    display: none;
+  .container {
+    max-width: 72.25rem;
+    margin: 0 auto;
+  }
+
+  .work-box {
+    max-width: 72.25rem;
+    height: 26.25rem;
+    border-radius: 1.5rem;
+    flex-wrap: nowrap;
+  }
+
+  .reverse {
+    flex-direction: row-reverse;
+  }
+
+  .work-box:nth-child(3) {
+    cursor: pointer;
+    border: 2px solid #a7aeff;
+    box-shadow: 1px 10px 16px rgb(64, 83, 252, 0.24);
+  }
+
+  .work-img img {
+    height: 100%;
+  }
+
+  .work-info {
+    height: 100;
+    width: auto;
+    margin: 15px;
   }
 
-  .mobile_menu {
+  .work-meta p {
+    font-size: 1.125rem;
+    line-height: 1.5rem;
+  }
+
+  .about-portfolio {
     display: flex;
-    justify-content: space-between;
+    flex-direction: row;
+  }
+
+  .about-section {
+    background-color: #fff;
+    border-top-right-radius: 100px;
+    margin-top: 40px;
+  }
+
+  .about-box {
+    width: 50%;
+    margin-left: 15px;
+    margin-top: 2rem;
+    margin-bottom: 2rem;
+  }
+
+  .lang-container {
+    width: 50%;
+    margin-top: 2.5rem;
+    margin-bottom: 2rem;
+    margin-right: 4rem;
+  }
+
+  .hand {
+    display: block;
+    position: absolute;
+    margin-left: 25%;
+    margin-top: -30px;
+    width: 50px;
+    height: 50px;
+  }
+
+  #btn:active {
+    background-color: #162ae3;
+  }
+
+  #btn:hover {
+    box-shadow: 20px 30px 40px rgb(64, 83, 252, 0.24);
+    background-color: #4053fc;
+    border-radius: 8px;
+  }
+
+  .about-lang {
+    display: flex;
+    flex-direction: row;
     align-items: center;
-    margin-right: 20px;
+    padding-left: 1px;
   }
 
-  h1.heading {
-    font-size: 2.5rem;
+  .learn {
     display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 7.5rem;
+    margin: 1px 8px;
+  }
+
+  .skill-item {
+    margin: 1px;
+    margin-top: 10px;
+    margin-left: -22px;
+  }
+
+  .skill-name {
+    margin-left: -18px;
+    margin-top: 12px;
+  }
+
+  .contact-container {
+    align-items: center;
+    width: auto;
+    height: auto;
+    border-top-left-radius: 100px;
+  }
```

Comment:
- [ ] Kindly apply the background image for the desktop section from the Figma design.
#### Figma 
![contact-me-bg-image-figma](https://user-images.githubusercontent.com/44057863/167259235-5da26812-24ae-454f-882c-b5d08354ee3c.png)

#### Imp
![contact-me-bg-image-impl](https://user-images.githubusercontent.com/44057863/167259242-28e66c1e-866e-4ec4-a9bd-50d8b7e990af.png)
l

Reply:
exactly the same image that I am using 
I downloaded the desktop template of the image